src/document_basic.c

```diff
@@ -256,15 +250,15 @@ int Document_LoadSchemaFieldJson(Document *doc, RedisSearchCtx *sctx, QueryError
     }
 
     size_t oix = doc->numFields++;
-    doc->fields[oix].path = rm_strdup(field->path);
-    doc->fields[oix].name = (field->name == field->path) ? doc->fields[oix].path
```

Comment:
Why is there no need for `path` handling?

Reply:
Short answer is that path member got removed from `DocumentField`
No one was really using it so it seemed redudant.
You need the path in order to get the value but here we are already in the part of obtaining the value, so it isn't needed.